docs/05-technical-reference/00-architecture/obsv-02-architecture-logging.md

```diff
@@ -13,12 +17,9 @@ title: Logging Architecture
 5. The Loki server processes the log data and stores it in the log store. The data is indexed based on the passed labels
 6. The user queries the logs using Grafana dashboards to analyze and visualize logs fetched and processed by Loki. Learn more about [accessing Grafana](../../04-operation-guides/security/sec-06-access-expose-grafana.md).
 
-## Telemetry Component
-
-![](./assets/obsv-configurable-logging-architecture.drawio.svg)
 
-Kyma's classic in-cluster logging features (1-6) are unchanged, but with the integration of Kyma's [telemetry component](./../../01-overview/main-areas/observability/obsv-04-telemetry-in-kyma.md), you can use additional functionality:
+With the integration of Kyma's [telemetry component](./../../01-overview/main-areas/observability/obsv-04-telemetry-in-kyma.md), you can use additional functionality:
```

Comment:
dont like the sentence here as well but it was there before (may be I am wrong here)

we use `you can use additional functionality:` and the numbering scheme is different.

Reply:
I wanted to re-work the stuff anyway as part of the tracing feature rollout. Either we drop the sentence fully or we keep it, I think there is nothing in between

Reply:
lets keep it the old way for now.